.gitconfig

```diff
@@ -0,0 +1,3 @@
+[branch]
```

Comment:
Please remove this file from the PR. If you want to set this option, best to do it in .git/config (your local repo config file)